vms/platformvm/camino_service.go

```diff
@@ -65,64 +65,67 @@ type GetBalanceResponseWrapper struct {
 
 func (response GetBalanceResponseWrapper) MarshalJSON() ([]byte, error) {
 	if !response.LockModeBondDeposit {
-		return json.Marshal(response.avax)
+		return stdjson.Marshal(response.avax)
 	}
-	return json.Marshal(response.camino)
+	return stdjson.Marshal(response.camino)
 }
 
 // GetBalance gets the balance of an address
 func (s *CaminoService) GetBalance(_ *http.Request, args *GetBalanceRequest, response *GetBalanceResponseWrapper) error {
-	s.vm.ctx.Log.Debug("Platform: GetBalance called")
+	s.vm.ctx.Log.Debug("Platform: GetBalance called",
+		logging.UserStrings("addresses", args.Addresses),
+	)
 
+	s.vm.ctx.Lock.Lock()
```

Comment:
All vm.state calls must be guarded by lock-unlock, but we must unlock right after, because of possible `s.Service.GetBalance` call, which will do its own lock.